beets/ui/commands.py

```diff
@@ -39,7 +39,7 @@
 from beets import library
 from beets import config
 from beets import logging
-from beets.util.confit import _package_path
```

Comment:
This has been importing the internal `_package_path` function. Now that confuse is a separate library this seems like the wrong thing to do. Either that functionality should be exposed publicly in confuse or we should find another way to do this.

Reply:
Indeed. The (sole) use of this could probably be easily replaced with a more standard `os.path.dirname(__file__)` sort of construction.

Reply:
Great, then I'll sort this small change out separately and we can merge this now!